src/interceptors/api-response-interceptor.js

```diff
@@ -11,15 +11,15 @@ import errorCodes from '../error-codes';
 * @returns {Object} - Returns the generated API
 */
 export default function apiResponseInterceptor(session, request, response) {
-  if (response.qHandle && response.qType) {
+  if (response && response.qHandle && response.qType) {
     return session.getObjectApi({
       handle: response.qHandle,
       type: response.qType,
       id: response.qGenericId,
       genericType: response.qGenericType,
     });
   }
-  if (response.qHandle === null && response.qType === null) {
+  if (response && response.qHandle === null && response.qType === null) {
```

Comment:
What is the expected shape of the response? If qHandle is undefined is that an error or can we have responses without those? 

I made it so they were errors, but then unit tests failed ... so I'm guessing `null` is relevant here.

Reply:
the integrations tests were failing due to an expired apiKey in env-vars - now fixed

I see no issue with your changes, but can you explain when that can happen, this is maybe a backend issue? AFAIK, the response should always be there

Reply:
The respone can be the be the value of result or the only out parameter. And therefore almost any type of value.
I think it is possible to have a method where a null response is a valid value but I don't think the engine has any

Reply:
> I think it is possible to have a method where a null response is a valid value but I don't think the engine has any
as @T-Wizard mentioned, afaik engine doesn't have any valid "null" responses

@chrismanley maybe it's worth checking with the engine team

(if none) we should catch & document this error with `createEnigmaError` and add a new entry in the [error-codes.js](https://github.com/qlik-oss/enigma.js/blob/master/src/error-codes.js) ex: `EMPTY_RESPONSE` or similar